operator/config/autopilot-channels/cluster-workload-identity/recorder_remove_hostport_patch.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- op: remove
+  path: /spec/template/spec/containers/0/ports/0/hostPort
```

Comment:
Mind adding some comment to explain what problem this solves? I have a vague impression that the Pod API has some issue about the defaulting values around port. But I cannot remember the details. I guess future maintainers may have the same question.

Reply:
This is copying existing patch scripts around to try to make them follow the normal kustomize structure...